client/create.go

```diff
@@ -43,3 +62,12 @@ func runCreate(cmd *Command, args []string) {
 	}
 	fmt.Printf("Created key with initial version %d\n", versionID)
 }
+
+func readDataFromStdin() []byte {
+	fmt.Println("Reading from stdin...")
+	data, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		fatalf("Problem reading key data: %s", err.Error())
+	}
+	return data
```

Comment:
I'd recommend returning the error in this function and handling that error in the runCreate function

Reply:
Thank you so much! I have revised my code according to your comment.